baseplate/integration/pyramid.py

```diff
@@ -62,6 +62,18 @@ def baseplate_tween(request):
     return baseplate_tween
 
 
+class BaseplateEvent(object):
+
+    def __init__(self, request):
+        self.request = request
+
+
+class ServerSpanInitialized(BaseplateEvent):
```

Comment:
could you add this to the pyramid integration docs with a note explaining when this is guaranteed to fire during request lifecycle please?